contracts/wooracle/WooracleV2_2.sol

```diff
@@ -260,57 +261,24 @@ contract WooracleV2_2 is Ownable, IWooracleV2 {
     }
 
     /// @notice the price decimal for the specified base token
-    function decimals(address) external pure override returns (uint8) {
+    function decimals(address) external pure returns (uint8) {
         return 8;
     }
 
-    function cloPrice(address _base) external view override returns (uint256 refPrice, uint256 refTimestamp) {
+    function cloPrice(address _base) external view returns (uint256 refPrice, uint256 refTimestamp) {
         return _cloPriceInQuote(_base, quoteToken);
     }
 
     function isWoFeasible(address _base) external view override returns (bool) {
         return infos[_base].price != 0 && block.timestamp <= (timestamp + staleDuration);
     }
 
-    function syncTS() external onlyAdmin {
-        timestamp = block.timestamp;
-    }
-
-    function syncTS(uint256 _ts) external onlyAdmin {
-        timestamp = _ts;
-    }
-
-    function debugTS()
-        external
-        view
-        returns (
-            uint256 n,
-            uint256 bs,
-            uint256 ts,
-            bool f
-        )
-    {
-        n = block.number;
-        bs = block.timestamp;
-        ts = timestamp;
-        f = block.timestamp <= (timestamp + staleDuration);
-    }
-
-    function woSpread(address _base) external view override returns (uint64) {
-        return infos[_base].spread;
-    }
-
-    function woCoeff(address _base) external view override returns (uint64) {
-        return infos[_base].coeff;
-    }
-
-    // Wooracle price of the base token
-    function woPrice(address _base) external view override returns (uint128 priceOut, uint256 priceTimestampOut) {
+    function woPrice(address _base) external view returns (uint128 priceOut, uint256 priceTimestampOut) {
```

Comment:
`woPrice` function call be deleted, it's able to get price by `woState`

Reply:
Fixed.

Reply:
Fixed.